resolwe_bio/descriptors/abyss_template.yml

```diff
@@ -1,7 +1,7 @@
 - name: assembler:abyss
   label: ABySS annotation
   description: ABySS assebler annotation template
-  var:
+  schema:
```

Comment:
Field in `DescriptorSchema` is named `schema` so we use same name here. Otherwise we should change both.